tests/post/steps/test_salt_api.py

```diff
@@ -48,7 +48,8 @@ def login_salt_api(host, username, password, version, context, request):
     ])
     with host.sudo():
         cmd_output = host.check_output(cmd_ip)
-    ip = json.loads(cmd_output)['local'][0]
+    # We take the last IP, otherwise we can have the control plane VIP instead
+    ip = json.loads(cmd_output)['local'][-1]
```

Comment:
Isn't it the opposite in Vagrant?

Reply:
Just checked, looks like it is...

Reply:
We should probably replace this whole way of retrieving the control-plane IP and use the grain `metalk8s.control_plane_ip` instead.

Reply:
smart